src/googledrive.js

```diff
@@ -93,6 +94,18 @@ function googleDrivePath (path) {
   return cleanPath(`${PATH_PREFIX}/${path}`);
 }
 
+/**
+ * Remove surrounding quotes from a string.
+ *
+ * @param {string} string - string with surrounding quotes
+ * @returns {string} string without surrounding quotes
+ *
+ * @private
+ */
+function removeQuotes (string) {
+  return string.substring(1, string.length-1);
```

Comment:
This is not actually removing just surrounding quotes, but the first and last character, no matter what it is. Not sure that's a good idea, at the very least not with this function name.